tools/rna_tools/vienna_rna/rnafold.xml

```diff
@@ -54,20 +73,77 @@
-                <param name="circular" type="boolean" truevalue="--circ" falsevalue="" checked="false" label="Assume circular RNA structure" help="--circ"/>
-            </when>
-        </conditional>
+        <section name="advancedOptions" title="Advanced options">
+            <param name="noconversion"
+                   type="boolean"
+                   truevalue=""
+                   falsevalue="--noconv"
+                   checked="true"
+                   label="Convert Thymine to Uracil (T -> U)"
+                   argument="--noconv"
+                   help="Avoids confusion with DNA sequences"/>
+
+            <param name="gquad"
+                   type="boolean"
+                   truevalue="--gquad"
+                   falsevalue=""
+                   checked="false"
+                   label="Take G Quadruplex formation into account"
+                   argument="-g"/>
+
+            <param name="nolp"
+                   type="boolean"
+                   truevalue=""
+                   falsevalue="--noLP"
+                   checked="true"
+                   label="Allow lonely pairs"
+                   argument="--noLP"/>
+
+            <param name="nogu"
+                   type="boolean"
+                   truevalue=""
+                   falsevalue="--noGU"
+                   checked="true"
+                   label="Include GU pairing"
+                   argument="--noGU"
+                   help="Allow wobbly pairs G-U."/>
+
+            <param name="noclosinggu"
+                   type="boolean"
+                   truevalue=""
+                   falsevalue="--noClosingGU"
+                   checked="true"
+                   label="Allow GU pairing at the ends"
+                   argument="--noClosingGU"
+                   help="Allow pairing of G and U at the ends of helices."/>
+
+            <param name="notetra"
+                   type="boolean"
+                   truevalue=""
+                   falsevalue="--noTetra"
+                   checked="true"
+                   label="Use stabilization for loops, hairpins etc."
+                   argument="--noTetra"/>
+
+            <param name="canonicalonly"
+                   type="boolean"
+                   truevalue=""
+                   falsevalue="--canonicalBPonly"
+                   checked="true"
+                   label="Allow non-canonical basepairings"
+                   argument="--canonicalBPonly"/>
+
+            <param name="circular"
+                   type="boolean"
+                   truevalue="--circ"
+                   falsevalue=""
+                   checked="false"
+                   label="Model as circular RNA structure"
+                   argument="--circ"/>
+        </section>
     </inputs>
     <outputs>
-        <data format="tabular" name="tabular_file"/>
+        <data format="dbn" name="tabular_file"/>
```

Comment:
This is awesome! But I guess we need to adopt the other wrappers as well to consume this filetype.